streams/src/test/resources/broker-defaults.properties

```diff
@@ -0,0 +1,34 @@
+# See http://kafka.apache.org/documentation.html#brokerconfigs for default values.
+
+# Each broker is uniquely identified by a non-negative integer id.  This id serves as the brokers "name", and allows
+# the broker to be moved to a different host/port without confusing consumers. You can choose any number you like so
+# long as it is unique.
+broker.id=0
+
+# Hostname of broker. If this is set, it will only bind to this address.  If this is not set, it will bind to all
+# interfaces, and publish one to ZK.
+host.name=127.0.0.1
+
+# The port on which the server accepts client connections.
+port=9092
+
+# The default number of partitions per topic.
+#
+num.partitions=1
+
+# Enable auto creation of topic on the server. If this is set to true then attempts to produce, consume, or fetch
+# metadata for a non-existent topic will automatically create it with the default replication factor and number of
+# partitions.
+auto.create.topics.enable=true
+
+# The maximum size of a message that the server can receive.  It is important that this property be in sync with the
+# maximum fetch size your consumers use or else an unruly consumer will be able to publish messages too large for
+# consumers to consume.
+#
+# Be careful with this setting when producing messages in batches with compression enabled.  In such a scenario the
+# batch of messages is treated as a single message, and its total size must be smaller than this setting.
+#
+message.max.bytes=1000000
+
+# Enable controlled broker shutdown.
+controlled.shutdown.enable=true
```

Comment:
Do we actually need this separate file? Why don't we just set this properties via `KafkaConfig`?